diff --git a/src/checkpoints.cpp b/src/checkpoints.cpp
index ce548fc..ba159b1 100644
--- a/src/checkpoints.cpp
+++ b/src/checkpoints.cpp
@@ -44,8 +44,11 @@ namespace Checkpoints
         ( 900000,   uint256("0x1ee5e72f281b50f95ac5452f9f2b1fca34595b99b8815a85a7ab97e964919860") )
         ( 972000,   uint256("0xd302a7015edd96078ba02a5d4b51e7377428386e3e911a76eb0dd5bddaf0697a") )
         ( 1117300,   uint256("0x190e3b4224643d868b451f2b8e13a2a48ac605dc776e661920fc1ab48e27a460") )
+        ( 1230333,   uint256("0xeba330f3efa967bef8f9e6530577a3e030b2e442cc817dce5460eb1beb5e1b20") )
+        ( 1330333,   uint256("0xf81d9e7013b95ea3be7d41674029dd6fe2c8f4abf6ecefb98c4a80ad01eeb5bc") )
     ;

+
     // TestNet has no checkpoints
     MapCheckpoints mapCheckpointsTestnet =
         boost::assign::map_list_of
diff --git a/src/net.cpp b/src/net.cpp
index adb2c76..a9eab40 100644
--- a/src/net.cpp
+++ b/src/net.cpp
@@ -27,7 +27,7 @@

 using namespace std;

-static const int MAX_OUTBOUND_CONNECTIONS = 16;
+static const int MAX_OUTBOUND_CONNECTIONS = 25;

 bool OpenNetworkConnection(const CAddress& addrConnect, CSemaphoreGrant *grantOutbound = NULL, const char *strDest = NULL, bool fOneShot = false);

